src/testFixtures/java/org/opensearch/neuralsearch/BaseNeuralSearchIT.java

```diff
@@ -291,6 +291,23 @@ protected void createPipelineProcessor(final String modelId, final String pipeli
         createPipelineProcessor(requestBody, pipelineName, modelId);
     }
 
+    protected void createPipelineProcessorWithoutModelId(final String requestBody, final String pipelineName) throws Exception {
```

Comment:
Can we combine this method with the method written below by using if else? It will reduce code redundancy

Reply:
Already combined